<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgFixFilepaths</class>
 <widget class="QDialog" name="DlgFixFilepaths">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fix File Paths</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Didn't find file with name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labFilename">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtOldname">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;b&gt;Please select file's directory:&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="edtNewname"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnSelect">
       <property name="text">
        <string>select file</string>
       </property>
       <property name="icon">
        <iconset resource="../lib/lib.qrc">
         <normaloff>:/lib/file_opendir.png</normaloff>:/lib/file_opendir.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnIgnore">
       <property name="text">
        <string>&amp;Ignore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>&amp;Cancel Fixing</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSelect</sender>
   <signal>clicked()</signal>
   <receiver>DlgFixFilepaths</receiver>
   <slot>selectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>784</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectFile()</slot>
 </slots>
</ui>
